packages/@coorpacademy-components/src/organism/brand-form/index.js

```diff
@@ -9,6 +9,22 @@ import Button from '../../atom/button';
 import Link from '../../atom/link';
 import style from './style.css';
 
+const RolesFormGroup = ({title, subtitle, fields}) => {
```

Comment:
Most part of the redesigned pages will use this... so it'll need to be refactored